### ISSUE TITLE
Add more steps to Qubes clean install (from R4.0 to R4.1)

### ISSUE THREAD

**Issue by Minimalist73** [QubesOS/qubes-issues#7678]
### The problem you're addressing (if any)
When you do a [clean install](https://www.qubes-os.org/doc/upgrade/4.1/#clean-installation), you have to restore a backup coming from Qubes R4.0. It means that all templates and standalones VMs are still using the old 4.0 repository.
When you use the `qubes-dist-upgrade` tool, a specific step is done to do those repository changes but, while doing a clean install, a user is not instructed to do it.

### The solution you'd like
Add a note or more steps to the clean install part which says that a user have to upgrade his templates/standalones VMs to the new repo.
Since `qubes-dist-upgrade` is not available on 4.1 (from what I've seen, could be wrong), a user could use the [following script](https://github.com/QubesOS/qubes-dist-upgrade/blob/release4.0/scripts/upgrade-template-standalone.sh) manually to upgrade everything or we could allow the installation of qubes-dist-upgrade in 4.1 and tell users to use it by doing `sudo qubes-dist-upgrade -l`

### The value to a user, and who that user might be
The value is huge and impact everyone doing clean install from 4.0. A lot of users will not get the new 4.1 Qubes packages which can lead to security issues.

**Comment by andrewdavidwong** [comment]
@marmarek: As far as I can recall, we've never had to tell users to manually change their *template* repo defs to a newer Qubes release. Is this already handled automatically some way, or do we really need some brand new documentation for this?

**Comment by Minimalist73** [comment]
> @marmarek: As far as I can recall, we've never had to tell users to manually change their _template_ repo defs to a newer Qubes release. Is this already handled automatically somehow, or do we really need some brand new documentation for this?

I would like to know too if that's handled automatically. When I made the switch on 4.1 Alpha I had to switch all my templates and standalones VM to 4.1 otherwise I was stuck with R4.0 core packages. Maybe an upgrade process has been added with the 4.1 release but I don't see any commit related to that matter on the backup/restore tool github repo. Maybe it's done elsewhere I don't know.
